slosh/Cargo.toml

```diff
@@ -4,6 +4,9 @@ version = "0.10.0"
 edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
```

Comment:
Do we need "default=[]' here?  Want to make sure these dips are not only optional but are off by default.  Some anyway, regex will probably become default on when we wrap it with forms.

Reply:
yes. that's much more clear. good catch.